apps/finance/app/src/App.js

```diff
@@ -96,7 +96,7 @@ class App extends React.Component {
         onResolve={this.handleResolveLocalIdentity}
         onShowLocalIdentityModal={this.handleShowLocalIdentityModal}
       >
-        <SyncIndicator visible={isSyncing} />
+        <SyncIndicator visible={isSyncing} shift={50} />
```

Comment:
We should bump the aragonUI’s version to get the new SyncIndicator :-)

Reply:
Oooh seems I have it linked 🤦‍♂️

Reply:
Wondering if we should apply this change in the other apps as well?